.github/ISSUE_TEMPLATE/submit-method-package.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Submit method package
+about: Form to submit a new or updated method package
+
+---
+
+Thank you for submitting a method package to the HCA DCP Methods Registry!
+
+To expedite your package's addition to https://data.humancellatlas.org/analyze/methods,
+please provide the following package metadata.
+
+Required:
+- Tool title: 
+- Contact name: 
+- Contact email: 
+- Who to attribute: 
+- Language: 
+- GitHub repository URL: (_e.g. https://github.com/yourusername/yourpackagename_)
+- Upstream registry URL: (_e.g. https://bioconductor.org/packages/release/bioc/html/slingshot.html_)
+- 2-3 sentence description:
+
```

Comment:
+ - Method-ready Docker image URL: (do we assume dockerfile in code repository or ask for URL?)
+ - URL for downloadable test data [reference test data set, reference output(s)]
+ - Documentation for command line usage (free-text box)
+ - Documentation for validation of installation (free-text box)

Reply:
@jlchang: Good idea.  Can you provide examples for those?  Not a blocker, but would be nice.

Reply:
We don't yet have a docker image URL for one of our registered methods. I have a link for a docker that the EBI group has made for sc3 (which is a registered method) BUT I have no idea if Martin, the methodologist, has looked at this docker image or not...
if we just want a valid URL from quay.io:
https://quay.io/repository/biocontainers/bioconductor-sc3-scripts
also, this link has the following comment:
No builds have been run for this repository
So I'm not sure we'd want to use it
The web page does generate this:
docker pull quay.io/biocontainers/bioconductor-sc3-scripts

Here are examples for STREAM, a not-yet-registered method:
URL for Method-ready Docker image: https://hub.docker.com/r/pinellolab/stream/
command line to pull Method-ready Docker image: docker pull pinellolab/stream

Dockerfile URL: https://github.com/pinellolab/STREAM/blob/master/Dockerfile
URL for downloadable test data, reference test data set(s): https://www.dropbox.com/sh/xnw9ro22bgrz2pa/AADQWmyCjUekg3hudvhsrAWka?dl=0
OR
 git clone https://github.com/pinellolab/STREAM.git
 cd STREAM/Datasets/
URL for downloadable test data, reference output(s): TBD
Documentation for command line usage (free-text box): https://github.com/pinellolab/STREAM#transcriptomic-data-1
Documentation for validation of installation (free-text box): TBD

Think I should reach out to the STREAM group if we're using links to their stuff as examples? Or wait until their method is registered?